scripts/build/Platform/Windows/build_config.json

```diff
@@ -134,7 +134,7 @@
       "CMAKE_OPTIONS": "-G \"Visual Studio 16 2019\" -DCMAKE_SYSTEM_VERSION=10.0 -DLY_TEST_IMPACT_INSTRUMENTATION_BIN=%TEST_IMPACT_WIN_BINARY%",
       "CMAKE_LY_PROJECTS": "AutomatedTesting",
       "CMAKE_TARGET": "ALL_BUILD",
-      "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo"
+      "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo /target:DisplayMessage"
```

Comment:
Will this execute the other msbuild targets or only the DisplayMessages target that was added?

Reply:
just a temp commit, will take it back, I want to see what Jenkins has for certain MSBuild variables